google-cloud-bigquerystorage/pom.xml

```diff
@@ -108,6 +108,12 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20200518</version>
```

Comment:
Please manage the version in `<dependencyManagement>` section of the root pom, instead of here. This is for long-term maintenance purpose.

Reply:
Got it.